src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -572,18 +572,25 @@ protected function getContainerBaseClass()
      */
     protected function initializeContainer()
     {
+        $asFiles = true;
         $class = $this->getContainerClass();
         $cacheDir = $this->warmupDir ?: $this->getCacheDir();
         $cache = new ConfigCache($cacheDir.'/'.$class.'.php', $this->debug);
         $oldContainer = null;
         if ($fresh = $cache->isFresh()) {
+            if (!$asFiles) {
```

Comment:
this cannot happen

Reply:
:) If I change the value line 575, it will happen.
But OK, I will add something to be able to change it in a clean manier. So 4.4, you win!
But I don't know how to do that with the DIC. I think the only solution is to add a new paratemer (the DIC is not build at this time)